html/elements/summary.json

```diff
@@ -52,6 +52,60 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "display: list-item": {
```

Comment:
`display: list-item` is not a good identifier. Remember that we want to be able to call this from macros. `{{compat("html.elements.summary.display: list-item")}}` wouldn't work I think. Please rename to `display_list_item` or similar.

Reply:
And you can add a description with this string, so that it will be displayed in the table.

Reply:
Got it. That'll be in the impending update to the PR; thanks.